Wbooks/src/app/screens/logIn/layout.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { TextInput, Button, Text, ImageBackground } from 'react-native';
+import PropTypes from 'prop-types';
+
+import { white } from '../../../constants/colors';
+
+import styles from './styles';
+
+function LogInScreen({ handleChange, handleBlur, handleSubmit, values }) {
+  return (
+    <ImageBackground style={styles.logInContainer} source={require('./../../assets/bc_inicio.png')}>
```

Comment:
You can import assets like any other import `import icon from './../../assets/bc_inicio.png'` and then do `source={icon}`